skietbaan_fe/src/components/CreateComp.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { Jumbotron, Form, Button } from 'react-bootstrap';
+import propTypes from 'prop-types';
+import { connect } from 'react-redux';
+import '../scss/createcomp.css';
+import { createcomp } from '../actions/competition.action';
+class CreateComp extends Component {
+	constructor() {
+		super();
+		this.state = { compTitle: '', compStatus: '' };
+		//binding the onChange method to this commponents
+		this.onChange = this.onChange.bind(this);
+		this.onSubmit = this.onSubmit.bind(this);
+	}
+	//------------------------------------------------------------------------
+	/** A method that detects the change in the change in thw textfield */
+	onChange(e) {
+		this.setState({ [e.target.name]: e.target.value });
+	}
+	//------------------------------------------------------------------------
+	/** A method that handles the submit enent for the submit button*/
+	onSubmit(e) {
+		/** Preventing the default button action event to occur automatically*/
+		e.preventDefault();
+		const compData = {
```

Comment:
Why is this variable a const?
Won't the title change later on?

Reply:
That compData, is an initialised competition object that will be transferred to createComp(compData)

Reply:
Great